roles/scp_http_int/tasks/http_int_credential.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Placing ssh public key for Jenkins nodes
+  authorized_keys:
+    user: logs
+    state: present
+    key: "{{ lookup('file', '../files/jenkins.pub') }}"
```

Comment:
Could we fetch it from the jenkins server rather than place it in the repo ?

Reply:
We can't. This sever is not connected to Jenkins server by design.

Reply:
No, but ansible will be able to connect to both. The idea is to use the delegate_to feature of Ansible.